core/core.go

```diff
@@ -653,7 +650,15 @@ func (t *Textile) threadInterceptor() grpc.UnaryServerInterceptor {
 		// Collect the user if we haven't seen them before.
 		user, ok := mdb.UserFromContext(ctx)
 		if ok && user.CreatedAt.IsZero() {
-			if err := t.collections.Users.Create(ctx, owner); err != nil {
+			var ffsInfo *mdb.FFSInfo
+			if t.powc != nil {
+				ffsId, ffsToken, err := t.powc.FFS.Create(ctx)
+				if err != nil {
+					return nil, err
+				}
+				ffsInfo = &mdb.FFSInfo{ID: ffsId, Token: ffsToken}
+			}
+			if err := t.collections.Users.Create(ctx, owner, ffsInfo); err != nil {
```

Comment:
Create an FFS instance for new `User`s

Reply:
Always forget which case was covering. It's the first gRPC call that some new user does, right?

Reply:
That is my understanding, yes. This interceptor always checks for an existing user. So if that user isn't found, it's the first time we've seen them, so we create the FFS instance and user record.